sdk/keyvault/azkeys/models.go

```diff
@@ -259,35 +245,62 @@ func keyItemFromGenerated(i *generated.KeyItem) *KeyItem {
 	}
 
 	return &KeyItem{
-		Attributes: keyAttributesFromGenerated(i.Attributes),
+		Attributes: keyPropertiesFromGenerated(i.Attributes),
 		KID:        i.Kid,
 		Tags:       convertGeneratedMap(i.Tags),
 		Managed:    i.Managed,
 	}
 }
 
-// DeletedKeyBundle - A DeletedKeyBundle consisting of a WebKey plus its Attributes and deletion info
-type DeletedKeyBundle struct {
-	KeyBundle
+// DeletedKey - A DeletedKey consisting of a WebKey plus its Attributes and deletion info
+type DeletedKey struct {
+	// The key management attributes.
+	Attributes *KeyProperties `json:"attributes,omitempty"`
+
+	// The Json web key.
+	Key *JSONWebKey `json:"key,omitempty"`
+
 	// The url of the recovery object, used to identify and recover the deleted key.
 	RecoveryID *string `json:"recoveryId,omitempty"`
 
+	// The policy rules under which the key can be exported.
+	ReleasePolicy *KeyReleasePolicy `json:"release_policy,omitempty"`
+
+	// Application specific metadata in the form of key-value pairs.
+	Tags map[string]string `json:"tags,omitempty"`
+
 	// READ-ONLY; The time when the key was deleted, in UTC
 	DeletedDate *time.Time `json:"deletedDate,omitempty" azure:"ro"`
 
+	// READ-ONLY; True if the key's lifetime is managed by key vault. If this is a key backing a certificate, then managed will
+	// be true.
+	Managed *bool `json:"managed,omitempty" azure:"ro"`
+
 	// READ-ONLY; The time when the key is scheduled to be purged, in UTC
 	ScheduledPurgeDate *time.Time `json:"scheduledPurgeDate,omitempty" azure:"ro"`
 }
 
 // DeletedKeyItem - The deleted key item containing the deleted key metadata and information about deletion.
 type DeletedKeyItem struct {
-	KeyItem
+	// The key management attributes.
+	Attributes *KeyProperties `json:"attributes,omitempty"`
```

Comment:
1. Should be `Properties`, but...
2. Why is `DeletedKeyItem` public? We don't expose this in other language SDKs, and not sure how it differs from a `DeletedKey` anyway.